BUILD.bazel

```diff
@@ -155,11 +157,11 @@ cc_test(
         "googletest/samples/sample7_unittest.cc",
         "googletest/samples/sample8_unittest.cc",
     ],
+    linkstatic = 0,
```

Comment:
Looks like there is no good reason to move this line from previous location.Could you please limit the changes to the actual factual changes to remove the noise.